docs/LSPApi.md

```diff
@@ -511,6 +511,53 @@ public class MyLSPSemanticTokensFeature extends LSPSemanticTokensFeature {
 }
 ```
 
+### Semantic Tokens File View Providers
+
+LSP4IJ can help incorporate a file's semantic tokens information so that it's readily available and can be used to
+implement specific behavior based on whether an element is a declaration or reference, string or numeric literal,
+comment, etc., based on the reported semantic tokens. This is implemented via a custom `FileViewProviderFactory` and
+`FileViewProvider`. LSP4IJ includes default implementations of these for TextMate files which otherwise lack PSI element
+structure, and it provides a simple way for other file types to gain access to the same features.
+
+#### LSPSemanticTokensFileViewProviderFactory
+
+Most files use a `SingleRootFileViewProvider`, and those that do can use `LSPSemanticTokensFileViewProviderFactory`.
+If specialized behavior is needed, `LSPSemanticTokensFileViewProviderFactory` subclassed and and implemented.

```

Comment:
```suggestion
If specialized behavior is needed, `LSPSemanticTokensFileViewProviderFactory` can be subclassed and implemented.

```

Reply:
Yup. That's what I get for moving fast with some changes. Thanks for the keen eye. I'll make all of these updates and push a new commit shortly.